### PR TITLE
288 hide some site health issues - Closes #288

### LINKED ISSUE #288: Hide Some Site Health Issues
There are some issues and recommended improvements in the WordPress Site Health Tool that we don't agree with and that we'd like to hide.

These include (but may not be limited to):

- You should use a persistent object cache
- Comments break into multiple pages
- Email domain issues detected: DMARC
- Have a default theme available

### PR DESCRIPTION
**Pull request by CNoemi94**
Hid **Persistent Object Cache and Comment Pagination**

### CODE REVIEW

File: includes/class-support-overrides.php [hunk 1]
```diff
@@ -49,6 +49,11 @@ public function __construct() {
 		if ( class_exists( 'WooCommerce' ) ) {
 			add_filter( 'woocommerce_background_image_regeneration', '__return_false' );
 		}
+		function remove_site_health_test( $tests ) {
+			unset( $tests['direct']['persistent_object_cache'], $tests['direct']['yoast-page-comments-check'] );
+			return $tests;
+		}
+		add_filter( 'site_status_tests', 'remove_site_health_test', 99 );
 	}
```
Comment (1337rout) [inline includes/class-support-overrides.php:57]: Can you please move this function to inside of the class as a static public function and then call the hook in this init function? I'm making the suggestion in github so linting and whitespace may not be perfect. Please don't use the commit suggestion feature and instead manually commit this after testing on your local. 
```suggestion
		add_filter( 'site_status_tests', array( $this, 'remove_site_health_test', 99 ) );
	}
			public static function remove_site_health_test( $tests ) {
			unset( $tests['direct']['persistent_object_cache'], $tests['direct']['yoast-page-comments-check'] );
			return $tests;
		}
```

**Review by 1337rout — CHANGES_REQUESTED** [review]
Just one comment about making the function a class member and using it that way :)

**Review by 1337rout — APPROVED** [review]
LGTM! Thanks for updating it. :sunglasses:

**Comment by blogtutor** [comment]
Looks good, tested on dev.nerdpress.net.  

Before:

image

After:

image

**Pull request merged**